root_s3_object_lifecycle.tf

```diff
@@ -39,4 +42,22 @@ locals {
         noncurrent_days = local.default_expiration_days
       }
   }]
+
+  non_prod_export_bucket_lifecycle_rules = [
+    {
+      id     = "delete-export-bucket-objects"
+      status = "Enabled"
```

Comment:
Make the `status` value a local set according to the environment, eg: `export_bucket_policy_status  = local.environment == "prod" ? "Disabled" : "Enabled"`